cogs/commands.py

```diff
@@ -268,6 +268,16 @@ async def scoreboard(self, ctx):
             table.append((str(user), user.messages_sent))
 
         await ctx.send(f'>>> ```prolog\n{tabulate(table, headers=("User", "Messages",), tablefmt="fancy_grid")}\n```')
+                       
+                       
+    @commands.command(aliases=['e'])
+    async def emoji(self, ctx, *, name):
+        """Send an emoji, useful for those who don't have nitro but want to send animated emojis"""
+        emoji = discord.utils.get(ctx.guild.emojis, name=name)
+        if emoji is not None:
+            await ctx.send(emoji)
+        else:
+            await ctx.send("Emoji not found")
```

Comment:
This is just something I like to do;

Instead of `if: else:` here you can do:
```python
if emoji is not None:
    return await ctx.send(emoji)
    
return await ctx.send("Emoji not found")
```